CONTRIBUTING.md

```diff
@@ -2,6 +2,45 @@
 
 You'd like to help? Great!  :tada:
 
+All contributions to `PyPRECIS` are made via merges with the develop branch.
```

Comment:
Do we want to change this to read `are made via merges with the main branch`?

Reply:
Yes we do - I will update this shortly - starting by getting main up to date with this PR